### PR TITLE
init external storage

### PR DESCRIPTION
**Pull request by Klaven**
Initial info on using external storage.

Not sure if I added the link to it in the right place. Let me know what you think of it as a start. we can add more later but I thought this covered our bases.

### CODE REVIEW

**Comment by jenting** [comment]
Seem like a little duplicate https://github.com/SUSE/doc-caasp/blob/master/adoc/admin-ses-integration.adoc#prerequisites, maybe we could enhance at here.

**Comment by kkaempf** [comment]
@jordimassaguerpla this needs to be prominently mentioned in the release notes of `4.0.1`

**Comment by jordimassaguerpla** [comment]
This is a P1 because it blocks the release of a P1 bug.

**Comment by Klaven** [comment]
@jenting :thinking: not sure. this is more of a warning that external storage might need you to install stuff. and that is a document on how to use a specific external storage.

File: adoc/admin-external-storage.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+= Configuring External Storage
+
+Because {productname} delivers the minimum packages required to support kubernetes not every package needed for every backing storage solution will be installed on the nodes. As each node is {sle}, you can use your already in place tools for deploying and managing packages.
+
+If using autoyast or terraform these packages can easily be installed when initializing the node.
+
+As an example, Ceph requires the folowing packages to be installed:
```
Comment (ereslibre) [inline adoc/admin-external-storage.adoc]:
```suggestion
As an example, Ceph requires the following packages to be installed:
```

**Comment by jordimassaguerpla** [comment]
Do Ceph requires nfs-client? It is not mentioned here https://github.com/SUSE/doc-caasp/blob/master/adoc/admin-ses-integration.adoc#prerequisites.

I would add a reference from https://github.com/SUSE/doc-caasp/blob/master/adoc/admin-ses-integration.adoc#prerequisites to this document.

I would be more explicit on how to install the packages on a node.

**Comment by r0ckarong** [comment]
> Do Ceph requires nfs-client? It is not mentioned here https://github.com/SUSE/doc-caasp/blob/master/adoc/admin-ses-integration.adoc#prerequisites.
> 
> I would add a reference from https://github.com/SUSE/doc-caasp/blob/master/adoc/admin-ses-integration.adoc#prerequisites to this document.
> 
> I would be more explicit on how to install the packages on a node.

I agree with jordi. @Klaven could you please add examples for autoyast, terraform and zypper to install these three packages for the example?

I think it would make more sense to actually convert this into the beginnings of a "software management" chapter. Since we need to describe how to install packages in the three ways before bootstrap (and after with zypper only).

Right now this is a generalized remark that some storage solutions might need additonal packages but that should be made clear in the respective instructions as Rafa rightly said.

File: adoc/admin-external-storage.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+= Configuring External Storage
+
+Because {productname} delivers the minimum packages required to support kubernetes not every package needed for every backing storage solution will be installed on the nodes. As each node is {sle}, you can use your already in place tools for deploying and managing packages.
+
+If using autoyast or terraform these packages can easily be installed when initializing the node.
```
Comment (r0ckarong) [inline adoc/admin-external-storage.adoc]: Could you please add examples for ay/tf/zypper how that would look and be used to install these packages? I can then rework this into a more generalized doc.

**Comment by r0ckarong** [comment]
Will rework to have this warning as an intro to "Integration -> External Storage".

**Comment by Klaven** [comment]
@r0ckarong here are some examples for how someone might install something.
terraform:
in their terraform.tfvars file, note I do not list all repositories you might need:
```# define the repositories to use
# EXAMPLE:
# repositories = {
#   repository1 = "http://example.my.repo.com/repository1/"
#   repository2 = "http://example.my.repo.com/repository2/"
# }
repositories = {
        ....
}

# Minimum required packages. Do not remove them.
# Feel free to add more packages
packages = [
  "kernel-default",
  "-kernel-default-base",
  "ca-certificates-suse",
  "xfsprogs",
  "ceph-common"
]```

And for the autoyast:
```
  
  <software>
    <image/>
    <products config:type="list">
      <product>SLES</product>
    </products>
    <instsource/>
    <patterns config:type="list">
      <pattern>base</pattern>
      <pattern>enhanced_base</pattern>
      <pattern>minimal_base</pattern>
      <pattern>basesystem</pattern>
    </patterns>
    <packages config:type="list">
      <package>glusterfs</package>
      <package>ksh</package>
    </packages>
  </software>
```

**Pull request merged**